modules/meteroid/src/api/coupons/service.rs

```diff
@@ -54,8 +54,8 @@ impl CouponsService for CouponsServiceComponents {
             expires_at: req.expires_at.map(chrono_from_timestamp).transpose()?,
             redemption_limit: req.redemption_limit,
             tenant_id,
-            recurring_value: -1, // todo fixme later
-            reusable: false,     // todo fixme later
+            recurring_value: None, // todo fixme later
+            reusable: false,       // todo fixme later
```

Comment:
this has been implemented no ?

Reply:
yes, I need to add them to proto so then I can map them here 👍

Reply:
fixed